tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -87,9 +89,9 @@ local utils = import 'templates/utils.libsonnet';
     },
     command: utils.scriptCommand(
       |||
-        %s 
-        %s 
-        %s 
+        %s
+        %s
+        %s
```

Comment:
The diff looks fine to me, but I didn't read through line-by-line. Since this should be causing most of the diff, should we move this out of the current PR to verify the resulting diff is empty?

Reply:
My editor must have removed these by accident. I checked each of the diffs, and the only change is this whitespace, plus the change I made to use gcloud ssh in the TF tests.